src/Bridges/Constraint/bridges/number_conversion.jl

```diff
@@ -120,3 +120,43 @@ function MOI.get(
     s = MOI.get(model, MOI.ConstraintSet(), bridge.constraint)
     return MOI.Utilities.convert_approx(S1, s)
 end
+
+function MOI.supports(
+    model::MOI.ModelLike,
+    attr::Union{MOI.ConstraintPrimalStart,MOI.ConstraintDualStart},
+    ::Type{NumberConversionBridge{T,F1,S1,F2,S2}},
+) where {T,F1,S1,F2,S2}
+    return MOI.supports(model, attr, MOI.ConstraintIndex{F2,S2})
```

Comment:
You should just return `true`, the `AbstractBridgeOptimizer` takes care of also asking `supports` downstream

Reply:
Hmm. I guess we're inconsistent about this. I should go through and check.